devops/automation/azure-pipelines.yml

```diff
@@ -1,12 +1,16 @@
 # Pipeline for Binding Tools for Swift.
 #
 # yaml help: https://aka.ms/yaml
-
 trigger:
-- '*'
+  branches:
+    include:
+    - '*'
 
 pr:
-- '*'
+  autoCancel: true
```

Comment:
Should we also have this in macios or *do we have this in the macios one?

Reply:
What do you mean?

Reply:
I'll message you!